dependencies.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+rm -rf dependencies_analysis
```

Comment:
probably better to do file handling in python; this script will just fail if the directory doesn't exist already